src/reducers/comments.tsx

```diff
@@ -112,12 +112,17 @@ export const createEmptyCommentInfo = (): CommentInfo => {
 };
 
 export type CommentKeyParams = {
+  commentId: number | undefined;
```

Comment:
Thanks for talking me though this one. I see now that there is a lot of functionality in `CommentInfo` that we need for editing so it does make sense to add `commentId` to the info key.

After supporting the edit case, the property `info.beginNewComment` seems a bit misleading. Maybe we can call it `info.showForm`, `info.beginCommentChange`, or something? Since this patch is probably going to get quite large, maybe hold off on renaming things. We could do it in follow-up issues, too.

Reply:
filed #1285